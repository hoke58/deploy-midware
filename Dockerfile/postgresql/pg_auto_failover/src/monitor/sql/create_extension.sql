-- Copyright (c) Microsoft Corporation. All rights reserved.
-- Licensed under the PostgreSQL License.

create extension pgautofailover cascade;
