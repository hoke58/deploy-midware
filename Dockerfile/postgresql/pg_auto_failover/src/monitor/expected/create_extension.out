-- Copyright (c) Microsoft Corporation. All rights reserved.
-- Licensed under the PostgreSQL License.
create extension pgautofailover cascade;
NOTICE:  installing required extension "btree_gist"
